packages/remirror__pm/package.json

```diff
@@ -185,6 +185,18 @@
   "publishConfig": {
     "access": "public"
   },
+  "@remirror": {
+    "tsconfigs": {
+      "src": {
+        "compilerOptions": {
+          "baseUrl": ".",
+          "paths": {
+            "prosemirror-tables": "node_modules/prosemirror-tables/src"
```

Comment:
Why do we need this? Is this related to the compilation error we saw before?

Reply:
That was the theory. But looks like it doesn't work.

I was trying to avoid using the `prosemirror-tables/dist` which contains the error we're seeing